### ISSUE TITLE
UGUI API Missing ？

### ISSUE THREAD

**Issue by Bian-Sh** [studentutu/Tilia.UnityUI#2]
Import this package via UPM git url .
Unity version : 2019.4.28f1c1
UGUI 1.0.0 see pic below

and unity has report these two error，you can see：

```
Library\PackageCache\io.extendreality.tilia.unityui@99f30b883b\Runtime\SharedResources\Scripts\VRTK3.3.0 UI\VRTK4_UIGraphicRaycaster.cs(169,61): error CS0117: 'GraphicRegistry' does not contain a definition for 'GetRaycastableGraphicsForCanvas'
```

```
Library\PackageCache\io.extendreality.tilia.unityui@99f30b883b\Runtime\SharedResources\Scripts\VRTK3.3.0 UI\VRTK4_UIGraphicRaycaster.cs(202,44): error CS1061: 'Graphic' does not contain a definition for 'raycastPadding' and no accessible extension method 'raycastPadding' accepting a first argument of type 'Graphic' could be found (are you missing a using directive or an assembly reference?)

```

how can i solve these problems，thks~

Unity UI Version：
![image](https://user-images.githubusercontent.com/24520547/146784240-753008be-34ca-4f7e-a5fd-e2e8fbd0eeca.png)

**Comment by studentutu** [comment]
@Bian-Sh  i believe it is a problem with the Unity 2019 (Verified from Unity 2021.3.x + ) -> I haven't verified it for the older versions. I will try however to fix this issue with a Unity definitions.

**Comment by studentutu** [comment]
@Bian-Sh  fixed in v 1.4.2  see here - https://github.com/studentutu/Tilia.UnityUI/releases/tag/v1.4.2

**Comment by Bian-Sh** [comment]
got you~ thks for your patch~
